### DISCUSSION TITLE
new TPMS sensors

### DISCUSSION THREAD

**Discussion by DavidIgor** [cedricp/ddt4all#1084] [Q&A]
all right, I have installed new tpms sensors in to my wheels and they won't sinc after I hit "TP set" on Dacia Dokker 2018 . 
Do I have to manually change wheel ID in DDT4ALL? 
new sensors are transmitting correctly and I have written down sensors IDs. screenshot is from original sensors....
![image](https://github.com/cedricp/ddt4all/assets/122531259/639a9b19-ae0b-4cfa-aad2-47bde0dc5e57)

**Comment by KarelSvo** [comment]
https://partsouq.com/en/catalog/genuine/unit?c=RENAULT201910&ssd=%24%2AKwGqno-F88bR7eDy2eWL-_LmxsHfrqGsrb-Qo-vt3srW0tDTloGtx-nt6djO18aAnZeqo9rw8_L77r-sodip4Pb1_t6s8P_moa6rp-b0-_K0rLz8_N7Sq6ytqYCxvv336_z3_fvOrqCn2NStqqus3-KvoKPq2a-rrq7g7KC9tvbt3tKrrODpoqC3_dnQrK2rrq3xsabx_d6q5u73roH95KyvrqitqK4AAAAAJyIASA%3D%3D%24&vid=0&cid=2041793456&uid=33736&q=

new original sensor works with TP set, without ddt4all
4 pieces 50 euros

**Comment by DavidIgor** [comment]
@KarelSvo  yeah, me new ones suppose to work too....  is there a way to write them manually?
